src/test/http_openai_handler_test.cpp

```diff
@@ -547,7 +673,7 @@ TEST_F(HttpOpenAIHandlerParsingTest, ParsingMessagesEmptyImageUrlFails) {
     doc.Parse(json.c_str());
     ASSERT_FALSE(doc.HasParseError());
     std::shared_ptr<ovms::OpenAIChatCompletionsHandler> apiHandler = std::make_shared<ovms::OpenAIChatCompletionsHandler>(doc, ovms::Endpoint::CHAT_COMPLETIONS, std::chrono::system_clock::now(), *tokenizer);
-    EXPECT_EQ(apiHandler->parseMessages(), absl::InvalidArgumentError("Url should contain base64 encoded string followed by \"base64,\" prefix or valid URL"));
+    EXPECT_EQ(apiHandler->parseMessages(), absl::InvalidArgumentError("Image file  parsing failed: can't fopen"));
 }
```

Comment:
I know it's in another scope but do we have a test for loading image from the network from non existing location?